storage/migrations/14_block_meta.up.sql

```diff
@@ -0,0 +1,19 @@
+BEGIN;
+
+ALTER TABLE chain.nodes
+    ADD COLUMN consensus_pubkey_address TEXT;
+
+CREATE INDEX ix_nodes_consensus_pubkey_address ON chain.nodes (consensus_pubkey_address);
+
+ALTER TABLE chain.blocks
+    ALTER COLUMN block_hash DROP NOT NULL,
+    ALTER COLUMN time DROP NOT NULL,
+    ALTER COLUMN num_txs DROP NOT NULL,
+    ALTER COLUMN namespace DROP NOT NULL,
+    ALTER COLUMN version DROP NOT NULL,
+    ALTER COLUMN type DROP NOT NULL,
+    ALTER COLUMN root_hash DROP NOT NULL,
+    ADD COLUMN proposer_node_consensus_pubkey_address TEXT,
+    ADD COLUMN signer_node_consensus_pubkey_addresses TEXT[];
```

Comment:
Heads up - we have a request to support queries for proposed/signed blocks for a given validator (https://app.clickup.com/t/86941rbrt), and I imagine we'll have to add indexes on these two columns to support that (gin index for the latter). Your call on whether you want to add that in this PR though.